### PR TITLE
CloudWatch Events Resource/Property

### PR DESCRIPTION
**Pull request by nmyster**
Added CloudWatch Events support based on new release of Cloudformation support for CloudWatch Events.

Using the Same namespace as Cloudformation (Events) rather than CloudWatchEvents as main python file

Adds Rule resource and Target Property

### CODE REVIEW

File: examples/CloudWatchEvents.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            }]
+        })],
+    AssumeRolePolicyDocument={"Version": "2012-10-17", "Statement": [
+        {
+            "Action": ["sts:AssumeRole"],
+            "Effect": "Allow",
+            "Principal": {
+                "Service": [
+                    "lambda.amazonaws.com",
+                    "apigateway.amazonaws.com"
```
Comment (svdgraaf) [inline examples/CloudWatchEvents.py]: You don't need this one ;)

Comment (nmyster) [inline examples/CloudWatchEvents.py]: haha thanks - i haven't contributed to this before so used your PR as a guide :)

Comment (svdgraaf) [inline examples/CloudWatchEvents.py]: Me neither ;)

Comment (nmyster) [inline examples/CloudWatchEvents.py]: well i guess good luck to both of us 👍

**Comment by markpeek** [comment]
Thanks for the quick implementation!

**Pull request merged**